### PR TITLE
Add Debezium input connector, fix field names in output connectors

### PR DESCRIPTION
**Pull request by Karakatiza666**
Fix https://github.com/feldera/feldera/issues/813
Fix https://github.com/feldera/feldera/issues/802

### CODE REVIEW

**Comment by ryzhyk** [comment]
I already tried it, and it worked for me.

**Review by ryzhyk — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,12 +7,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Add Debezium input connector
+  ([#813](https://github.com/feldera/feldera/issues/813))
 
 ### Fixed
 
 - Fixes a regression in the CSV parser
   ([#801](https://github.com/feldera/feldera/pull/801))
+- Fix field names in output connectors
+  ([#802](https://github.com/feldera/feldera/issues/802))
+- Made connector type icons look nicer in Pipeline Builder
```
Comment (gz) [inline CHANGELOG.md:23]:
```suggestion

### Enhancements
- Made connector type icons look nicer in Pipeline Builder
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,12 +7,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+### Added
+
+- Add Debezium input connector
+  ([#813](https://github.com/feldera/feldera/issues/813))
 
 ### Fixed
 
 - Fixes a regression in the CSV parser
   ([#801](https://github.com/feldera/feldera/pull/801))
+- Fix field names in output connectors
```
Comment (gz) [inline CHANGELOG.md:21]:
```suggestion
- Clarify some label names for output connectors
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,12 +7,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+## [0.1.5] - 2023-10-10
+
+### Added
+
+- Add Debezium input connector
+  ([#813](https://github.com/feldera/feldera/issues/813))
 
 ### Fixed
 
 - Fixes a regression in the CSV parser
```
Comment (gz) [inline CHANGELOG.md:19]:
```suggestion
- Fixes a regression in the CSV parser where it rejected the last row as an invalid row if `\r\n` was used for line-endings
```

**Review by gz — APPROVED** [review]

**Pull request merged**